code/modules/power/lighting/light.dm

```diff
@@ -176,6 +174,8 @@
 		var/color_set = bulb_colour
 		if(color)
 			color_set = color
+		if(reagents)
+			START_PROCESSING(SSmachines, src)
```

Comment:
Small question I had in my dreams, why do we set it to process when we get reagents? Don't we only want it to get hot while the light switch is on, in which case it would process itself anyways?

Reply:
From my understanding, process was only used to charge a depleted emergency cell. The light isn't actually running process() when it is on unless there's a cell that's being charged inside (or, in this PR, is heating reagents). Regardless, removing these lines stops the lights from heating under normal circumstances.